dist/index.html

```diff
@@ -15,28 +15,48 @@
   <body>
     <header>
       <div class="menu-bar">
-        <p class="logo" font-size="24pt">Time Trippers</p>
+        <h1 class="logo" font-size="24pt">Time Trippers</h1>
         <p class="user-name-display" id="tripper-name"></p>
         <p class="year-trip-cost" id="tripCost"></p>
       </div>
     </header>
-    <main>
+      <h2>Login Page</h2><br>
+      <div class="login" id="logIn">
+        <form id="login" method="get" action="login.php">
+          <label><b>username
+          </b>
+          </label>
+          <input type="text" name="Uname" id="Uname" placeholder="Username">
+          <br><br>
+          <label><b>password
+          </b>
+          </label>
+          <input type="Password" name="Pass" id="Pass" placeholder="Password">
+          <br><br>
+          <input type="button" name="log" id="log" value="Log In Here">
+          <br><br>
+          <input type="checkbox" id="check">
+          <span>Remember me</span>
+          <br><br>
+          Forgot <a href="#">Password</a>
```

Comment:
this empty anchor tag might ding you in the lighthouse audit. Maybe a <p> tag would fix that?